doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,58 @@
 
 <h3>New features since last release</h3>
 
+* Added a quantum debugger (`PLDB`) which interfaces via `qml.breakpoint()` and provides tools for 
```

Comment:
pldb?

Reply:
When referring to the PennyLane Debugger as a whole I use capital (PLDB), when referring to the prompt specifically, I use lowercase (pldb).